docs/docs/concepts/policies.mdx

```diff
@@ -161,27 +144,9 @@ A flow can be started in several ways:
 - One flow can ["link" to another flow](../concepts/flows.mdx#link), which will initiate
   the linked flow and return to the original flow once the linked flow
   completes.
-- Flows, which are conversational patterns (e.g. correction, resume flow), are
-  automatically initiated by the `FlowPolicy` when needed.
-
-### Interruptions and Digressions
-
-Flows and their specifications should ideally model the happy path of a
-conversation. The `FlowPolicy` depends on other components to identify
-digressions and interruptions and interrupt the flow accordingly. For details on
-which digressions and interruptions the flow policy can handle, refer to
-[Conversation Repair](./conversation-repair.mdx).
-
-For example, the user message _"oh sorry I meant John"_ is a correction to a
-prior input. The `FlowPolicy` depends on another component, like the
-[LLM Command Generator](../concepts/dialogue-understanding.mdx), to identify it as a
-correction and set the corresponding slots. Based on this, the `FlowPolicy`
-initiates the correction flow.
+- Flows can be automatically added by the `FlowPolicy` in the case of 
```

Comment:
I would highlight that these do not tend do be user-defined business logic flows but shorter meta flows that smooth out the conversation